dockstore-common/src/main/java/io/dockstore/common/RegressionTest.java

```diff
@@ -16,7 +16,7 @@
 package io.dockstore.common;
 
 /**
- * Setting aside tests that require our confidential data bundles
+ * Tests with older dockstore client with new webservice
```

Comment:
Define older and new. 
Imagine you're a new developer on the project and you're an axe murderer, frustrated with trying to figure out what is going on.